config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,25 @@
+# This patch inject a sidecar container which is a HTTP proxy for the 
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+apiVersion: apps/v1
```

Comment:
why do we need this side card? what is it doing for us?

Reply:
The kub-rbac-proxy sidecar is protecting access to the operator metrics via doing SubjectAccessReviews against the apiserver. I.e. the prometheus serviceaccount should be the only user accessing these metrics. Till now we only secure the data on transit access by exposing metrics only via TLS, but we don't do authorization.

Reply:
@periklis then we should keep?

Reply:
Yes, we should keep the kube-rbac-proxy.